app/src/main/kotlin/edu/stanford/bdh/engagehf/messages/MessagesHandler.kt

```diff
@@ -30,6 +30,7 @@ class MessagesHandler @Inject constructor(
         val actionResult = messagesActionMapper.map(action = message.action)
         var failure = actionResult.exceptionOrNull()
         when (val messagesAction = actionResult.getOrNull()) {
+            is MessagesAction.NoAction -> Unit
```

Comment:
it should not be needed as it is covered in the else case (applies also in case you change the response to nullable instead of result)

Question: Should we complete the message below if we could not map any action for it?

Reply:
I'm not sure actually, would need to check iOS

Reply:
I checked: iOS only calls dismissMessage, if an action has been performed and the message has `isDismissible` set to true.